### DISCUSSION TITLE
Is there a link for your discord server?

### DISCUSSION THREAD

**Discussion by Raj2032** [gluesql/gluesql#1069] [General]
Just wondering if you guys have a link to your discord server?

**Comment by Raj2032** [comment]
Please disregard this I found it:
[gluesql discord server](https://discord.com/invite/C6TDEgzDzY)

I was wondering if you guys could put the link on your main page or something?

**Reply by tristan-f-r** [reply]
It already is on [the readme (check badges)](https://github.com/gluesql/gluesql), but I do agree that it should be more obvious.

**Reply by tristan-f-r** [reply]
I've made a PR (#1122) to resolve this.
